setup.py

```diff
@@ -37,6 +37,7 @@
       install_requires=[
           "numpy>=1.6.0",
           "pytools",
+          "packaging>=20.0",
```

Comment:
```suggestion
          "packaging>=20.0;python_version>='3.10'",
```
? Not sure that's worth it..

Reply:
If it works everywhere (and it seems to), then why not go with the wave of the future? But thanks for raising the question!